test/WebJobs.Extensions.Tests/Extensions/DocumentDB/Trigger/CosmosDBTriggerAttributeBindingProviderTests.cs

```diff
@@ -98,31 +88,29 @@ public static IEnumerable<ParameterInfo[]> GetParameters()
                     new[] { GetFirstParameter(type, "Func1") },
                     new[] { GetFirstParameter(type, "Func2") },
                     new[] { GetFirstParameter(type, "Func3") },
-                    new[] { GetFirstParameter(type, "Func4") },
-                    new[] { GetFirstParameter(type, "Func5") },
-                    new[] { GetFirstParameter(type, "Func6") }
+                    new[] { GetFirstParameter(type, "Func4") }
                 };
             }
         }
 
         private static class ValidCosmosDBTriggerBindigs
```

Comment:
After doing it I discovered that the test failed in AppVeyor since there was no environment or setting available. Is that right?